region/src/main/java/org/motechproject/nms/region/domain/State.java

```diff
@@ -35,6 +41,10 @@
     private List<District> districts;
 
     @Field
+    @Cascade(delete = true)
+    @Persistent(table = "nms_states_join_circles", mappedBy = "states", defaultFetchGroup = "true")
+    @Join(column = "state_id")
+    @Element(column = "circle_id")
     private List<Circle> circles;
```

Comment:
@larubbio  Do we just need this change (lines 44-47) and the change in LanguageServiceImpl.java?